Issue #1255 | Expanding of the benefits of the Company Accounts (#1270)
magento/merchdocs: src/customers/account-companies.md

title: Company Accounts
---

Magento 2 Company Account transforms a default [individual] ({% link customers/account-create.md %}) customer account to the company accounts and supports customers to create and manage company accounts.

> By introducing B2B company accounts to your Magento 2 store, you can simplify the corporate shopping, let companies create multiple sub-accounts with flexible permissions based on user roles in the organization. Depending on the customer groups of the company, the store owner can adjust promotions and prices to suit their needs. It allows admins to create highly customized offers to cater to the shoppers’ demands and get more orders.

Reviewer: ```suggestion
When you incorporate B2B company accounts in your Magento store, you can simplify the corporate shopping by enabling companies to create multiple sub-accounts with flexible permissions based on user roles in their organization. Depending on the customer groups of the company, a store administrator can adjust promotions and prices to suit their needs, and create highly customized offers that cater to the shoppers’ demands and increase orders.
```

Author: Done ✅

> Advantage of Magento 2 Company Account:

Reviewer: ```suggestion
Advantages of  a company account:
```

Author: Done ✅

> 1. [B2B]({% link configuration/general/b2b-features.md  %}) Company Account offers unlimited users and the creation of additional accounts, thereby simplifying corporate purchases.

Reviewer: This should really be an unordered list
```suggestion
1. - A [B2B]({% link configuration/general/b2b-features.md  %}) company account offers unlimited users and the creation of additional accounts, thereby simplifying corporate purchases.
```

> 1. There is an ability to build a smart B2B company account hierarchy with different [roles and permissions] ({% link customers/account-company-roles-permissions.md %}) as placing orders.

Reviewer: ```suggestion
- Includes support for a "smart" company account hierarchy with different [roles and permissions] ({% link customers/account-company-roles-permissions.md %}) for placing orders.
```

Author: Done ✅

> 1. Allows to increase income by offering a [Company Store Credits] ({% link customers/account-dashboard-company-credit.md %}) payment method.

Reviewer: ```suggestion
- Provides a mechanism for merchants to increase income by offering [company store credit] ({% link customers/account-dashboard-company-credit.md %}) as a payment method.
```

Author: Done ✅

> 1. Permit [managing] ({% link customers/account-company-manage.md %}) all Company Accounts in Magento Admin.

Reviewer: ```suggestion
1. Supports the [management] ({% link customers/account-company-manage.md %}) of all company accounts in the Admin.
```

Author: Done ✅

## View company accounts

> The Companies grid lists all active company accounts and pending requests, regardless of status setting. It also provides the tools needed to create and [manage]({% link customers/account-company-manage.md %}) company accounts. Use the standard [grid controls]({% link stores/admin-grid-controls.md %}) to filter the list, and adjust the [column layout]({% link stores/admin-grid-layout.md %}). For a list of column descriptions, see the Column Descriptions in [Managing Company Accounts]({% link customers/account-company-manage.md %}).

Reviewer: ```suggestion
The _Companies_ grid lists all active company accounts and pending requests, regardless of status setting. It also provides the tools needed to [create]({% link customers/account-company-create.md %}) and [manage]({% link customers/account-company-manage.md %}) company accounts. Use the standard [grid controls]({% link stores/admin-grid-controls.md %}) to filter the list, and adjust the [column layout]({% link stores/admin-grid-layout.md %}). For a list of column descriptions, see the _Column Descriptions_ section in [Managing Company Accounts]({% link customers/account-company-manage.md %}).
```

Author: Done ✅

> Company accounts can be set up from the storefront or from the Admin. By default, the ability to create company accounts from the storefront is enabled. If allowed in the configuration, a visitor to the store can request to open a company account. After the company account is approved, the company administrator can set up the company structure and users with various levels of permission.

Reviewer: ```suggestion
Customers can create a new company accounts from the storefront, or a merchant can create one from the Admin. By default, the ability to create company accounts from the storefront is enabled. If allowed by the configuration, a visitor to the store can request to open a company account. After the company account is approved, the company administrator can set up the company structure and users with various levels of permission.
```

Author: Done ✅
